### DISCUSSION TITLE
React 17 JSX Transform

### DISCUSSION THREAD

**Discussion by pooya121** [TypeStrong/ts-node#1704] [Q&A] [answered]
Hi. is there any way that I can enable react 17 JSX transforms in ts-node?

**Comment by cspotcode** [comment, marked as answer]
ts-node delegates all transformation tasks to the underlying compiler, be it typescript or swc.  Do typescript and/or swc support react 17 JSX transforms?  If so, is there a particular configuration that is necessary to enable that type of transformation?
